src/main/java/com/company/Cursor17/Util/LaptopFactory.java

```diff
@@ -0,0 +1,44 @@
+package com.company.Cursor17.Util;
+
+import com.company.Cursor17.Model.Laptop;
+
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class LaptopFactory {
+    private static final String[] brands = new String[]{"Lenovo", "Acer", "Dell"};
```

Comment:
I would suggest you use collections) apply it everywhere

Reply:
please, check 2nd commit, is it ok?